### DISCUSSION TITLE
some terrible problems

### DISCUSSION THREAD

**Discussion by BeautyLaTeX** [James-Yu/LaTeX-Workshop#3127] [Q&A]
`Error during making tmpdir to build TeX files. Please check the environment variables, TEMP, TMP, and TMPDIR on your system.'
'command 'latex-workshop.onEnterKey' not found'
'command 'latex-workshop.build' not found`
When I used this plug-in, I encountered a lot of problems, the above tips are just a few. I guarantee that I have configured the compilation chain correctly. So please solve thes
![Snipaste_2021-01-05_14-28-04](https://user-images.githubusercontent.com/38369557/103616471-79493a00-4f67-11eb-8ec0-7b292ea2938a.png)
e problems, I really like vscdoe, thank you!

**Comment by jlelong** [comment]
Please post the complete logs of the extension. Do not truncate or alter them.

![74528024-87b2ef80-4f6a-11ea-9c5d-4f9feff25524](https://user-images.githubusercontent.com/2910140/77959270-98c77e00-72d6-11ea-8c58-4934a9fde39a.png)

**Comment by BeautyLaTeX** [comment]
How can i get the complete logs?

**Comment by tamuratak** [comment]
> command 'latex-workshop.build' not found

> Error during making tmpdir to build TeX files. Please check the environment variables, TEMP, TMP, and TMPDIR on your system.

Because the environment variables `TEMP`, `TMP`, and `TMPDIR` on your system are not correct, LaTeX Workshop can not be activated. Fix your configuration.

**Reply by OliverLeenders** [reply]
what would a correct value for these variables be?

**Comment by jlelong** [comment]
From the TeX badge on the left panel as in the screenshot. If the TeX is not there, it means that the extension is not properly installed. Then, completely remove and reinstall it.

But, first fill the bug report template, it is crucial to identify issues.

**Comment by BeautyLaTeX** [comment]
I have already reinstalled th plg-ins compeletly again and again,but it is no use.So base on your view,I do not know how to fix the configuration.I try to check the win10 environment,the texlive and the vscode's path are correct,and then i have no idea.

**Comment by tamuratak** [comment]
See https://www.computerhope.com/issues/ch000549.htm.
